projects/aca-playwright-shared/src/page-objects/components/dataTable/data-table.component.ts

```diff
@@ -52,6 +52,9 @@ export class DataTableComponent extends BaseComponent {
   emptyListTest = this.getChild('adf-custom-empty-content-template');
   paginationButton = this.page.locator('.adf-pagination__block button').nth(0);
   paginationOptions = this.page.locator('#cdk-overlay-0 button');
+  sitesVisibility = this.page.locator('[class$="adf-datatable-body"] [data-automation-id*="datatable-row"] [aria-label="Visibility"]');
+  sitesName = this.page.locator('[class$="adf-datatable-body"] [data-automation-id*="datatable-row"] [aria-label="Name"]');
```

Comment:
Do you need [class$="adf-datatable-body"]? Can't it be just .adf-datatable-body?

Reply:
I dont need it, good point